orsopy/fileio/data_source.py

```diff
@@ -32,8 +35,8 @@ class Experiment(Header):
 
     title: str
     instrument: str
-    start_date: str
-    probe: Union[Literal["neutrons", "x-rays"]]
+    start_date: datetime
+    probe: Literal["neutrons", "x-rays"]
```

Comment:
The list of probes in Nexus is (from https://manual.nexusformat.org/classes/base_classes/NXsource.html?highlight=probe)

probe: (optional) [NX_CHAR](https://manual.nexusformat.org/nxdl-types.html#nx-char)
type of radiation probe (pick one from the enumerated list and spell exactly)
Any of these values:
- neutron
- x-ray
- muon
- electron
- ultraviolet
- visible light
- positron
- proton

Reply:
Replaced neutrons and x-rays with the values here. We don't see the use of the other radiation types at the moment.